stanford-earth/stanford_earth_helper
# EARTH 1197: Optimize Our Community images on HP

# READY FOR REVIEW
- (Edit the above to reflect status)

# Summary
- TL;DR - what's this PR for?
Swap out our community mosaic images for optimized versions

# Needed By (Date)
- When does this need to be merged by?

# Steps to Test

1. View the Our Community section at the bottom of the homepage and verify that the images are not broken and look good.

# Affected versions or modules
- Does this PR impact any particular module, version, or pull request?

# Associated Issues and/or People
- JIRA ticket SE3:-
- Other PRs
- Any other contextual information that might be helpful (e.g., description of a bug that this PR fixes, new functionality that it adds, etc.)
- Anyone who should be notified? (`@mention` them here)

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
